kombu/async/http/curl.py

```diff
@@ -218,9 +218,6 @@ def _setup_request(self, curl, request, buffer, headers, _pycurl=pycurl):
             if request.proxy_username:
                 setopt(_pycurl.PROXYUSERPWD, '{0}:{1}'.format(
                     request.proxy_username, request.proxy_password or ''))
-        else:
-            setopt(_pycurl.PROXY, '')
```

Comment:
Can you please explain this change? It seems unrelated to this PR.

Reply:
Sorry you are correct. That is a change for another PR. I branched from the wrong commit. I will fix that so this PR is only about this one issue.